src/components/Onboarding/Screens/GardenTypeSelector.js

```diff
@@ -37,102 +41,141 @@ function GardenTypeSelector() {
         <h1
           css={`
             ${textStyle('title1')};
+            font-size: 3em;
             margin-bottom: ${3 * GU}px;
           `}
         >
-          Garden type
+          Garden Type
         </h1>
         <div>
           <p
             css={`
-              ${textStyle('body1')};
+              ${textStyle('title4')};
               color: ${theme.contentSecondary};
             `}
           >
-            Choose which type of garden you'd like to launch.
+            Choose the type of garden you'd like to launch.
           </p>
         </div>
       </div>
       <div
         css={`
           display: flex;
           align-items: center;
-          justify-content: space-around;
-          margin-bottom: ${6 * GU}px;
+          justify-content: center;
         `}
       >
         <Card
           icon={defaultGardenLogo}
-          paragraph="Launch garden with new token"
+          paragraph="Create a new token that is native to your garden. "
           onSelect={handleSelectNative}
           selected={selectedType === NATIVE_TYPE}
-          title="Native"
+          animationData={nativeAnimation}
+          title="Native Token"
         />
         <Card
           icon={defaultGardenLogo}
-          paragraph="Launch garden with existing token"
+          paragraph="Use an existing ERC-20 token to within your garden."
           onSelect={handleSelectBYOT}
           selected={selectedType === BYOT_TYPE}
-          title="BYOT"
+          animationData={byotAnimation}
+          title="Pre-existing Token"
         />
       </div>
     </div>
   )
 }
 
-function Card({ icon, title, onSelect, paragraph, selected }) {
+function Card({
+  icon,
+  title,
+  onSelect,
+  paragraph,
+  selected,
+  animationData,
+  animationSegments,
+}) {
+  const [isPlaying, setIsPlaying] = useState(false)
+
   const theme = useTheme()
+
+  const handleStartAnimation = useCallback(() => {
+    setIsPlaying(true)
+  }, [setIsPlaying])
+
+  const handleStopAnimation = useCallback(() => {
+    setIsPlaying(false)
+  }, [setIsPlaying])
+
   return (
     <div
       css={`
         cursor: pointer;
-        padding: ${5 * GU}px;
+        padding-left: ${3.5 * GU}px;
+        padding-right: ${3.5 * GU}px;
+        padding-bottom: ${3.5 * GU}px;
+        margin: 10px;
+        max-width: 264px;
```

Comment:
Think this should be `274` instead

218px from child div + ((28px padding) * 2)
```suggestion
        max-width: 274px;
```